### ISSUE TITLE
Error: Index creation failed - after adding API URL & Key

### ISSUE THREAD

**Issue by robbiecren07** [styxlab/next-cms-ghost#168]
Issue:
![Capture](https://user-images.githubusercontent.com/6932895/153791824-75bcb1eb-c760-47b1-ba8d-af9a42e25ffe.PNG)

Steps I took:
I cloned the git repo, yarn install > yarn run dev - successfully opened localhost:300. 
After that I signed up for a free trail of Ghost Pro, then I created a custom integration within Ghost Admin to generate a key. Then I changed the CMS_GHOST_API_URL & CMS_GHOST_API_KEY on lines 5-6 in /lib/processEnv.ts - to the new key and URL which I generated. I saved the file and did yarn run dev, once I opened localhost:300, I was given the error.

Here is the console error as well:
![Capture](https://user-images.githubusercontent.com/6932895/153792094-298256ba-d084-45dc-9e5b-3121b8195167.PNG)

After the error, I also tried creating a .env.local file in the root with my CMS_GHOST_API_URL & CMS_GHOST_API_KEY.

**Comment by styxlab** [comment]
Welcome @robbiecren07! Can you publish your API URL and keys here (you can invalidate them after a day), so I have a chance to debug the issue on my end?

**Comment by robbiecren07** [comment]
Sure!
`https://l2-code.ghost.io`
`00704d0b67c452250f5086d7d7`

**Comment by styxlab** [comment]
Just noticed that error I am getting is different from yours:
![image](https://user-images.githubusercontent.com/19638263/153924739-47145d52-309d-43fc-a37d-4b5dc7fe84ee.png)

**Comment by styxlab** [comment]
The error I am getting has been reported here: https://github.com/vercel/next.js/issues/33860 and needs to be fixed upstream. Another workaround is to downgrade next.js to version `12.0.0`. I'll update this package once the upstream fix is available.

**Comment by styxlab** [comment]
@robbiecren07: I am still puzzled how you got the "Index creation failed" error. I cannot reproduce it.

**Comment by robbiecren07** [comment]
@styxlab Sorry for the delayed reply, when I get online later tonight I will try the cache control and downgrade nextjs and see how that works. 

As for the index error, I may have done something while I was trying to fix it. So maybe I'll do a clean git clone again and make sure I don't audit fix anything and just enter my API key and URL and see if I still get the index error. I'll report back later tonight.

**Comment by robbiecren07** [comment]
Ok I think I figured it out, which it's a big error on my end. I tried the cache control and downgrade on nextjs and that didn't work. So I created a new clone and only did npm install and switched out the API key and URL in `processEnv.ts` - I ran npm run dev and it opened successfully.

So I went back into my first repo, I realized I still had the `.env.local` which I created after trying to troubleshot, so I deleted that file and npm run dev worked. I think what happen was when I first changed out the API keys & URL, for the URL I left the forward slash at the end of URL. which is what caused the first error, then while trying to debug it I created the `.env.local` file, which just through the same error.

So in the end, no need to downgrade nextjs, or disable the cache control. Just needed to delete the `.env.local` and make sure to type the correct URL without the trailing forward slash in the `processEnv.ts` file.
